### ISSUE TITLE
How to create new Websharper.MVU project?

### ISSUE THREAD

**Issue by DejanMilicic** [dotnet-websharper/mvu#7]
It may sound like a trivial question, but if you go over documentation, Templates, etc  
there is no clear answer - how to create a new empty MVU project you can start working on?

If there is not a specific template for MVU applications, which of the existing 6 templates would be a good starting point?

**Comment by granicz** [comment]
You are right, there is no empty MVU project template per se among the current WebSharper templates, the closest thing to go from would be the SPA template (aka "bundle" project). This template has a marked client-side entrypoint and an optional server-side. See the [TodoMVC test project](https://github.com/dotnet-websharper/mvu/tree/master/WebSharper.Mvu.TodoMvc) in this repo for an actual example.
